OrangeHRMTests/PageObjects/Modules/TopbarMenu.cs

```diff
@@ -0,0 +1,24 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+
+namespace OrangeHRMTests.PageObjects.Modules
+{
+    public static class TopbarMenu
```

Comment:
I would better not to make it static, just create instance in BasePage and use it across pages

Reply:
Fixed